README.md

```diff
@@ -1 +1,37 @@
 
+For this example ill be using Management Services, which is a logical group of services that are deployed on management hubs (Previously known as Management Hub Components).
+
+2 Repos will coexist:
+- Preprod Management Service Set
+- Prod Management Service Set
+
+ServiceSet - A repo template for Argo ApplicationSet which deploys a set of services defined in values.yaml to a specific cluster.
+
+A service is a helm chart reference defined by:
+- serviceName
+- repoURL
+- targetRevision
+- path
```

Comment:
Add code snippet or template example for clarity, this can be interpreted by readers with little knowledge in multiple ways (ambigious)

Reply:
Added `services` example